deploy-pipeline.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+fly -t lite set-pipeline -p ras-deploy -c concourse/ras-deploy.yml  --load-vars-from concourse/secrets.yml
```

Comment:
Could just list this in the readme rather than provide a script for it (though is doesn't hurt)